Content.Shared/Radio/Components/SharedRadioJammerComponent.cs

```diff
@@ -0,0 +1,32 @@
+using Robust.Shared.Serialization;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.RadioJammer;
+
+[NetworkedComponent]
+
+public abstract partial class SharedRadioJammerComponent : Component
+{
+}
```

Comment:
Why is the component split between client and server? It looks like you can just have one shared component.